SwiftKit/src/main/java/org/swift/swiftkit/SwiftValueWitnessTable.java

```diff
@@ -285,15 +281,8 @@ public static MemorySegment initializeWithCopy(SwiftAnyType type, MemorySegment
 
         var mh = initializeWithCopy.handle(type);
 
-        try (var arena = Arena.ofConfined()) {
-            // we need to make a pointer to the self pointer when calling witness table functions:
-            MemorySegment indirectDest = arena.allocate(SwiftValueLayout.SWIFT_POINTER);
-            MemorySegmentUtils.setSwiftPointerAddress(indirectDest, dest);
-            MemorySegment indirectSrc = arena.allocate(SwiftValueLayout.SWIFT_POINTER);
-            MemorySegmentUtils.setSwiftPointerAddress(indirectSrc, src);
-
-            var returnedDest = (MemorySegment) mh.invokeExact(indirectDest, indirectSrc, wtable);
-            return returnedDest;
+        try {
+            return (MemorySegment) mh.invokeExact(dest, src, wtable);
```

Comment:
This `initializeWithCopy` change was not included in #236. But I realized this has the same issue with `destroy`. No need for extract indirection anymore.

Reply:
Makes sense